utils/cloud_constants.py

```diff
@@ -86,5 +86,6 @@
 OSA_API_SERVER_PORT = os.environ.get("OSA_API_SERVER_PORT", 5000)
 OSA_API_SERVER_URL = 'http://{host}:{port}'.format(host=OSA_API_SERVER_HOST, port=OSA_API_SERVER_PORT)
 DATA_INSERT_CONCURRENCY = int(os.environ.get("DATA_INSERT_CONCURRENCY", 10))
+STRING_CHAR_LIMIT = int(os.environ.get("STRING_CHAR_LIMIT", 2000))
```

Comment:
Because we have so many constants related to API server, I think it will be better to maintain all separately in a new `osa_api_constants` module. Cloud constants was originally intended for S3 related constants (I was the one who had started using this nomenclature 2 years ago in f8a ML projects 😅, so at the time that was my intention). Not necessary if it is too much work but you can use the IDEs refactor functionality for this quite easily as right now these are all referenced in only one module.

Reply:
This constant will be used while saving the data into csv file in inference, not related to osa_api :) If you think need to separate out, probably we can create `other_constants` module and move this as well as api related constant there. What you think @rootAvish ?

Reply:
Moved API and other non-relevant constant to `other_constants`